### PR TITLE
bugs fix; add iob_m_wire.vh iob_s_wire.vh

### PR DESCRIPTION
**Pull request by JDLopes**

### CODE REVIEW

File: hardware/include/iob_m_wire.vh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+`IOB_WIRE(valid_o, 1)        //Request valid.
```
Comment (jjts) [inline hardware/include/iob_m_wire.vh:1]: internal circuit wires do not need  _i and _o

.a_i(a) 
.b_o(b)

I want to know that a is connected to an instance output but the wires  itself does not specify direction: only portas and port maps

This is different for TB wires: there either drive (reg) or are driven (wire) 

I recommend using _r for drivers and _w for driven wires

File: hardware/include/iob_s_wire.vh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+`IOB_WIRE(valid_i, 1)        //Request valid.
```
Comment (jjts) [inline hardware/include/iob_s_wire.vh:1]: No

Comment (jjts) [reply hardware/include/iob_s_wire.vh:1]: we need iob_wire no sufffix

iob_tb_m_wire (tb is master) (_r for master bus drivers, _w for master bus response wires)
 iob_tb_s_wire (tb_is slave) (_r for slave bus response drivers, _w for slave bus driven writes)

File: hardware/include/iob_m_wire.vh (hunk 1 above)
Comment (JDLopes) [reply hardware/include/iob_m_wire.vh:1]: I wanted to save one include by connecting signals from _swreg_gen.vh directly to axil2iob instance. I will fix it!

**Review by jjts — COMMENTED** [review]
You forgot _v and _w for tb wires

**Comment by JDLopes** [comment]
> You forgot _v and _w for tb wires

Done!

**Pull request merged**